src/edit/Toolbar.vue

```diff
@@ -1,24 +1,53 @@
 <template>
-  <v-toolbar
-    height="72"
-    color="transparent"
-    class="elevation-0">
+  <v-toolbar height="72" color="transparent" class="elevation-0">
```

Comment:
The Toolbar component is also much simplified with https://github.com/ExtensionEngine/tce-sprout-video/pull/3/files#diff-c31caae4002cde1610e57406724be0233346402b6ab29f403e6da13536104c6bR2